### ISSUE TITLE
[EXPO] cannot find 'RCTMGLSwiftLog' in scope

### ISSUE THREAD

**Issue by erwanlpfr** [rnmapbox/maps#3070]
Hello, 

First, thank you for your work. 
I'm trying to set a build with expo/mapbox.
By doing a simple install/build and run with Expo 49, I get these errors.

```
Expo 49
rn-mapbox in latest with mapbox implem
```

```bash
bunx expo prebuild --clean
bunx expo run:ios 
```

```bash
❌  (/.../node_modules/@rnmapbox/maps/ios/RCTMGL-v10/RCTLog.swift:2:3)

  1 | func RCTMGLLogError(_ message: String, _ file: String=#file, _ line: UInt=#line) {
> 2 |   RCTMGLSwiftLog.error(message, file: file, line: line)
    |   ^ cannot find 'RCTMGLSwiftLog' in scope
  3 | }
  4 | 
  5 | func RCTMGLLogWarn(_ message: String, _ file: String=#file, _ line: UInt=#line) {


❌  (/...//node_modules/@rnmapbox/maps/ios/RCTMGL-v10/RCTLog.swift:6:3)

  4 | 
  5 | func RCTMGLLogWarn(_ message: String, _ file: String=#file, _ line: UInt=#line) {
> 6 |   RCTMGLSwiftLog.warn(message, file: file, line: line)
    |   ^ cannot find 'RCTMGLSwiftLog' in scope
  7 | }
  8 | 
  9 | func RCTMGLLogInfo(_ message: String, _ file: String=#file, _ line: UInt=#line) {


❌  (/...//node_modules/@rnmapbox/maps/ios/RCTMGL-v10/RCTLog.swift:10:3)

   8 | 
   9 | func RCTMGLLogInfo(_ message: String, _ file: String=#file, _ line: UInt=#line) {
> 10 |   RCTMGLSwiftLog.info(message, file: file, line: line)
     |   ^ cannot find 'RCTMGLSwiftLog' in scope
  11 | }
  12 | 
  13 | func RCTMGLLog(_ message: String, _ file: String=#file, _ line: UInt=#line) {


❌  (/...//node_modules/@rnmapbox/maps/ios/RCTMGL-v10/RCTLog.swift:14:3)

  12 | 
  13 | func RCTMGLLog(_ message: String, _ file: String=#file, _ line: UInt=#line) {
> 14 |   RCTMGLSwiftLog.log(message, file: file, line: line)
     |   ^ cannot find 'RCTMGLSwiftLog' in scope
  15 | }
  16 | 
  17 | func RCTMGLLogTrace(_ message: String, _ file: String=#file, _ line: UInt=#line) {


❌  (/...//node_modules/@rnmapbox/maps/ios/RCTMGL-v10/RCTLog.swift:18:3)

  16 | 
  17 | func RCTMGLLogTrace(_ message: String, _ file: String=#file, _ line: UInt=#line) {
> 18 |   RCTMGLSwiftLog.trace(message, file: file, line: line)
     |   ^ cannot find 'RCTMGLSwiftLog' in scope
  19 | }
```

# app.config.js
```typescript
const IS_DEV = 'development';

export default {
  expo: {
    name: IS_DEV ? 'app (dev)' : 'app',
    slug: 'app',
    privacy: 'hidden',
    experiments: {
      typedRoutes: true,
    },
    scheme: 'app',
    userInterfaceStyle: 'automatic',
    splash: {
      backgroundColor: '#ddffe2',
      image: './assets/splash.png',
      resizeMode: 'contain',
    },
    icon: './assets/icon.png',
    android: {
      package: IS_DEV ? 'com.app.dev' : 'com.app.app',
      permissions: ['ACCESS_BACKGROUND_LOCATION'],
    },
    ios: {
      bundleIdentifier: IS_DEV ? 'com.app.dev' : 'com.app.app',
      infoPlist: {
        UIBackgroundModes: ['location', 'fetch'],
      },
    },
    web: {
      bundler: 'metro',
    },
    owner: 'me',
    plugins: [
      [
        '@rnmapbox/maps',
        {
          RNMapboxMapsImpl: 'mapbox',
          RNMapboxMapsDownloadToken:
            'sk..-',
        },
      ],
      ['sentry-expo'],
      ['expo-localization'],
    ],
  },
};

```

I don't use any exotic libraries apart Reanimated 3.
The rest looks like an expo starter pack.

Thank you for your work ! 👍🏻

**Comment by mfazekas** [comment]
Hi, thanks for the report! To be able to look into setup related issues we need full steps to reproduce the issue. 
Ideally n a brand new project, in the form of
```
react-native init sample --version react-native@0.67.5
cd sample
npm install @rnmapbox/maps --save
cd ios
pod installl
...
react-native run-ios
```

closing the issue now, will reopen when you provide the steps to reproduce.
If you cannot reproduce with a  brand new project, please start a new project with the same rn version you have and compare package.json for different versions.

Please note that your issue is considered a [support request](https://github.com/rnmapbox/maps/wiki/Support) and we don't provide free [support](https://github.com/rnmapbox/maps/wiki/Support).

See also https://github.com/rnmapbox/maps/issues/2213

**Comment by erwanlpfr** [comment]
Seems to be fixed with : [10.1.0-beta.5](https://www.npmjs.com/package/@rnmapbox/maps/v/10.1.0-beta.5)!
Thanks!
